docs/root/intro/version_history.rst

```diff
@@ -77,6 +77,7 @@ Version history
 * rbac network filter: a :ref:`role-based access control network filter <config_network_filters_rbac>` has been added.
 * rest-api: added ability to set the :ref:`request timeout <envoy_api_field_core.ApiConfigSource.request_timeout>` for REST API requests.
 * router: added ability to set request/response headers at the :ref:`envoy_api_msg_route.Route` level.
+* router: addded ability to set attempt count in upstream requests, see :ref:`envoy_api_field_route.VirtualHost.include_request_attempt_count`.
```

Comment:
nit: s/addded/added.

And also this introduces warning, 

```
Warning, treated as error:
/../generated/rst/intro/version_history.rst:80:undefined label: envoy_api_field_route.virtualhost.include_request_attempt_count (if the link has no caption the label must precede a section header)
```

Probably something like the following might work:

```
* router: added ability to set attempt count in upstream requests, see :ref:`virtual host's include request attempt count flag <envoy_api_field_route.VirtualHost.include_request_attempt_count>`.
```